lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -745,15 +769,21 @@ states:
     nestedJourney: ADDRESS_AND_FRAUD
     exitEvents:
       next:
-        targetState: CRI_HMRC_KBV_M2B
+        targetState: CRI_DWP_KBV_J7
         checkIfDisabled:
-          hmrcKbv:
-            targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE_M2B
+          dwpKbv:
+            targetState: CRI_HMRC_KBV_M2B
+            checkIfDisabled:
+              hmrcKbv:
+                targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE_M2B
       enhanced-verification:
-        targetState: CRI_HMRC_KBV_M2B
+        targetState: CRI_DWP_KBV_J7
```

Comment:
Annoyingly I think we need to expand this out with its own `CRI_DWP_KBV_M2B` state and requisite events for the M2B-specific dropout and mitigation routes